api/app/controllers/spree/api/base_controller.rb

```diff
@@ -3,9 +3,6 @@
 module Spree
   module Api
     class BaseController < ActionController::Base
-      prepend_view_path Rails.root + "app/views"
-      append_view_path File.expand_path("../../../app/views", File.dirname(__FILE__))
```

Comment:
Is this removable because Api::BaseController now inherits from ActionController::Base instead of ActionController::Metal?  i.e. this no longer applies? https://github.com/jhawthorn/solidus/commit/546dd1852364d1e86214792eddec85b3764f82d5